.gitignore

```diff
@@ -51,6 +51,7 @@ npm-debug.log
 /globals.yml
 /levelbuilder_ci_last_run
 /locals.yml
+/log
```

Comment:
For cleaner history / blame, please pull this out into its own PR. You can assign me as a reviewer and merge immediately. 👍